### PR TITLE
frontend post

### PR DESCRIPTION
**Pull request by hougasian**

### CODE REVIEW

File: _posts/2019-03-06-welcome-new-hire-getting-started-in-frontend-at-rimdev.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Conferences are great opportunities to learn, be inspired, or inspire others by speaking! [VueConf](http://www.vueconf.us/) and [An Event Apart](https://aneventapart.com/) have become 2 of our top choices to date.
+
+### Writing
+
+As a developer or designer, you're faced with unique challenges every day -- so are your peers. [Write about them!](https://rimdev.io) Share your solution with the community. None of us work in a vacuum, and how you approach a solution through code or design can help everyone.
+
+### Open Source
+
+We love Open Source, from [things we've made](https://rimdev.io/open-source/), to contributing to community projects and [Hacktoberfest](https://hacktoberfest.digitalocean.com/). Everyone on the team uses their personal GitHub account to build your community footprint. Some of our projects are private, although we keep many public where we can so your contributions are yours.
```
Comment (khalidabuhakmeh) [inline _posts/2019-03-06-welcome-new-hire-getting-started-in-frontend-at-rimdev.md]:
```suggestion
We love Open Source, from [things we've made](https://rimdev.io/open-source/), to contributing to community projects and [Hacktoberfest](https://hacktoberfest.digitalocean.com/). Everyone on the team uses their personal GitHub account to build your community footprint. Some of our projects are private, although we keep much public in an effort to show your contributions.
```

File: _posts/2019-03-06-welcome-new-hire-getting-started-in-frontend-at-rimdev.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+### Ongoing
+
+Frameworks everywhere and all the things changing. Our landscape changes constantly, so we adapt and learn all the things. Instructor led training, subscriptions, whatever you need to succeed; [Pluralsight](https://www.pluralsight.com/), [egghead.io](https://egghead.io/), [Vue Mastery](https://www.vuemastery.com/), all there for the asking.
```
Comment (khalidabuhakmeh) [inline _posts/2019-03-06-welcome-new-hire-getting-started-in-frontend-at-rimdev.md]:
```suggestion
Our professional landscape changes constantly, so we adapt and learn all the things. We all learn differently so we make many options available: Instructor-led training, book reimbursements, subscriptions, whatever you need to succeed. Some examples of learning tools include [Pluralsight](https://www.pluralsight.com/), [egghead.io](https://egghead.io/), [Vue Mastery](https://www.vuemastery.com/).
```

**Comment by hougasian** [comment]
thanks @alexoragz

File: _posts/2019-03-06-welcome-new-hire-getting-started-in-frontend-at-rimdev.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Conclusion
+
+If being a part of a diverse, transparent, growing team while growing your career is what you're seeking, [Ritter Insurance Marketing's](https://ritterim.com) development team is a great environment. Team members to help you grow and face new challenges! If you're new the team, then Welcome! If you'd like to find out more, reach out on [twitter](https://twitter.com) 😃.
```
Comment (khalidabuhakmeh) [inline _posts/2019-03-06-welcome-new-hire-getting-started-in-frontend-at-rimdev.md]:
```suggestion
If being a part of a diverse, transparent, and growing team while growing your own career is what you're seeking, [Ritter Insurance Marketing's](https://ritterim.com) development team is a great environment. Team members are here to help you grow and face new challenges! If you're new the team, then Welcome! If you'd like to find out more, reach out to any of us on [Twitter](https://twitter.com) 😃and if you're new, don't be afraid to start a conversation with your new team members.
```

File: _posts/2019-03-06-welcome-new-hire-getting-started-in-frontend-at-rimdev.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+image_credit: "@ilumire"
+---
+
+New to the frontend at RIMdev, or looking for more about our Frontend team? Here's a quick overview of our team, tools, and culture!
+
+As a whole, RIMdev has always strived to use the technology that bets suites the task. So while this will give the current overview, software changes and so do we.
+
+The majority of our frontend team are Mac users, so this post will seem mac-biased, but rest assured, that's where it stops. Use whatever platform that makes you happy.
+
+Some of this content may be mirrored in @buhakmeh's post [Welcome New Hire: Getting Started At RIMdev
```
[the remark this answers is not in the document]
Comment (hougasian) [reply _posts/2019-03-06-welcome-new-hire-getting-started-in-frontend-at-rimdev.md]: @mentions converts the twitter link automatically

**Pull request merged**